src/pages/edge/lifecycle-for-edge-network/behavior-reference.md

```diff
@@ -0,0 +1,376 @@
+
+* [Lifecycle Application Launch (Foreground)](../../home/base/mobile-core/lifecycle/event-reference.md#lifecycle-application-foreground)
+* [Lifecycle Application Close (Background)](../../home/base/mobile-core/lifecycle/event-reference.md#lifecycle-application-background)
+
+For full event details, please read the [Lifecycle event reference](../../home/base/mobile-core/lifecycle/event-reference.md).
+
+#### Data payloads
+
+The Lifecycle for Edge Network event payloads are defined by the Platform Mobile Lifecycle Details XDM field group, and include information about the application, device, and environment when the event occurred. Please refer to [Lifecycle for Edge Network data](metrics.md) for the data included with these events.
+
+As a consequence of this shift in not relying on a session timeout window to consider new sessions, the potential Lifecycle states are greatly simplified:
+
+### Consecutive Lifecycle API calls
+
+Consecutive `lifecycleStart` and consecutive `lifecyclePause` API calls will not update their respective timestamps stored in persistence, and Lifecycle data is not changed.
+
+#### Consecutive `lifecycleStart` API calls
+
+![](./assets/index/lifecycle-start-after-start.svg)
+
+<!-- 
+%%{
+  init: {
+    'themeVariables': {
+      'lineColor': '#7a7a7a'
+    }
+  }
+}%%
+graph LR
+    graph1 ==> graph2
+
+    subgraph graph1 [" "]
+        direction TB
+        B(1a.<br/><code>lifecycleStart</code>) ==> C("1b.<br/>Application Launch (Foreground)")
+        class graph1 transparentSubgraph;
+    end
+
+    subgraph graph2 [" "]
+        direction TB
+        E(2a.<br/><code>lifecycleStart</code>) ==> F("2b.<br/>Call ignored, no event dispatched")
+        class graph2 transparentSubgraph;
+    end
+
+    classDef regularBox fill:#009c3b,stroke:#009c3b,color:#fff;
+    classDef incorrectBox fill:#EB1000,stroke:#EB1000,color:#fff;
+    classDef transparentSubgraph fill:transparent,stroke:#7a7a7a;
+
+    class B,C,F regularBox;
+    class E incorrectBox;
+ -->
+
+#### Consecutive `lifecyclePause` API calls
+
+![](./assets/index/lifecycle-pause-after-pause.svg)
+
+<!-- 
+%%{
+  init: {
+    'themeVariables': {
+      'lineColor': '#7a7a7a'
+    }
+  }
+}%%
+graph LR
+    graph1 ==> graph2
+
+    subgraph graph1 [" "]
+        direction TB
+        B(1a.<br/><code>lifecyclePause</code>) ==> C("1b.<br/>Application Close (Background)")
+        class graph1 transparentSubgraph;
+    end
+
+    subgraph graph2 [" "]
+        direction TB
+        E(2a.<br/><code>lifecyclePause</code>) ==> F("2b.<br/>Call ignored, no event dispatched")
+        class graph2 transparentSubgraph;
+    end
+
+    classDef regularBox fill:#009c3b,stroke:#009c3b,color:#fff;
+    classDef incorrectBox fill:#EB1000,stroke:#EB1000,color:#fff;
+    classDef transparentSubgraph fill:transparent,stroke:#7a7a7a;
+
+    class B,C,F regularBox;
+    class E incorrectBox;
+ -->
+
+## Expected Lifecycle scenarios
```

Comment:
I'd suggest to swap the ordering of this section with the previous "Consecutive Lifecycle API Calls" so the expected scenarios are listed first.